pkg/config/initoptions.go

```diff
@@ -11,6 +11,10 @@ import (
 	"strings"
 	"time"
 
+	"sigs.k8s.io/kustomize/kyaml/yaml"
```

Comment:
We should organize the imports into two groups. One for imports from the golang sdk and one for all other imports. This also applies to some of the other files in the PR.

Reply:
removed.